dev_guide/getting_traffic_into_cluster.adoc

```diff
@@ -430,7 +430,7 @@ endif::[]
 == Using a Service NodePort
 
 Use xref:../architecture/core_concepts/pods_and_services.adoc#service-nodeport[NodePorts] to expose the service nodePort on all nodes in the cluster.
-NodePorts are in the 30-60k range by default, which means a NodePort is
+NodePorts are in the 30k-60k range by default, which means a NodePort is
```

Comment:
The range is used when the nodeport is not specified. The user can specify the desired nodeport (or just use the default)